cypress/integration/group1/dashboard.ts

```diff
@@ -43,4 +43,24 @@ describe('Dockstore dashboard', () => {
       .should('have.attr', 'href')
       .and('include', 'getting-started-with-services');
   });
+  it('Registering new tool through Github redirects correctly', () => {
+    cy.visit('/dashboard?newDashboard');
+    cy.contains('Tools');
+    cy.get('#registerToolButton').should('be.visible').click();
+    cy.get('#3-register-workflow-option').should('be.visible').click();
```

Comment:
1. You're copying an unfortunate existing pattern we have everywhere, but ideally, you should be using `data-cy` instead of `id` to select in Cypress. See [Cypress best practices](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements). While we don't need to go back and fix everywhere where we're using `id`, we should avoid introducing new usages of `id` as a Cypress selector, if possible.
2. The `#3` is fragile (if we change the order), although the tests will catch it. If you switch to `data-cy` you may be able to come up with a better solution.

Reply:
changed from using `id` to `data-cy`